cves/kernel/CVE-2020-27777.yml

```diff
@@ -241,8 +246,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: rtas
```

Comment:
What is rtas? Can you provide a description in the note section?

I give this 3 upvotes.